src/cli.ts

```diff
@@ -4,6 +4,7 @@ import * as Bluebird from 'bluebird'
 import chalk from 'chalk'
 import { all as clearCachedModules } from 'clear-module'
 import { CommandNotFoundError, find, MissingRequiredArgsError, run as unboundRun } from 'findhelp'
+import { decode } from 'jsonwebtoken'
```

Comment:
Is this a new dependency? Do we need to put it in `package.json`?

Reply:
Dependency was already there